data/sponsorshipLevels.yml

```diff
@@ -1,97 +1,84 @@
+      - Paddock booth area for use each race weekend.
       - Special announcements over the PA throughout race weekends.
-      - Large advertisement in race program each round.
-      - Four season gate passes for paddock access.
+      - Promotional mention through WMRRA social media highlighting special promotions as requested. 
       - Four passes to the WMRRA Annual Awards Banquet.
-      - Paddock booth area available for use each race weekend.
-      - Display of sponsor-supplied banners each race weekend.
-      - Business description on WMRRA.com sponsors page.
-      - Logo and business link on WMRRA.com homepage.
-      - Promotional mentions on WMRRA social media, highlighting special promotions as requested.
 
-  - level: Race Event Sponsor
-    costInDollars: 1_000
-    benefits:
-      - Exclusive naming rights to a WMRRA race event (ex. WMRRA Vintage Day sponsored by YOU)
-      - Paddock booth area available for use the weekend of the event.
-      - Regular PA announcements during the race weekend.
-      - Medium advertisement in Race Program during the sponsored race event.
-      - Six gate passes good for the day of the event.
-      - Display of sponsor-supplied banners the day of the event.
-      - Logo and business link on WMRRA.com homepage.
-      - Promotional mentions on WMRRA social media.
-  
   - level: Race Class Sponsor
-    costInDollars: 500
+    tier: silver
+    costInDollars: 750
     benefits:
-      - Exclusive naming rights to a WMRRA race event (ex. Open Supersport sponsored by YOU)
+      - Exclusive naming rights to one of WMRRA’s race classes. (Ex. Open Super Sport sponsored by ABC)
       - Sponsor name on daily and year-end class championship trophies.
-      - PA announcements during the race weekend.
-      - Small advertisement in Race Program.
-      - Two season gate passes for paddock access.
+      - Sponsor logo on website sponsors page; https://wmrra.com/sponsor 
+      - Paddock booth area for use each race weekend in designated vendor area.
+      - PA announcements during the sponsored race event.
+      - Promotional mention through WMRRA social media.
       - Two passes to the WMRRA Annual Awards Banquet.
-      - Paddock booth area available for use each race weekend.
-      - Logo and business link on WMRRA.com homepage.
-      - Promotional mention on WMRRA social media.
-  
+    disclaimer: Some classes cost a higher amount, typically due to running multiple heats in a weekend.
+
   - level: General Club Sponsor
+    tier: bronze
     costInDollars: 500
     benefits:
-      - Small advertisement in Race Program.
-      - Two season gate passes for paddock access.
-      - Two passes to the WMRRA Annual Awards Banquet.
-      - Paddock booth area available for use each race weekend.
-      - Logo and business link on WMRRA.com homepage.
-      - Promotional mention on WMRRA social media.
+      - Sponsor logo on website sponsors page; https://wmrra.com/sponsor 
```

Comment:
suggestion: As discussed, if these could be actual hotlinks (e.g. [sponsors page](https://wmrra.com/sponsor)),that would be much more webby and professional. If it's too hard to make them into links, you should probably just drop the web addresses.

Reply:
![](https://media2.giphy.com/media/ChpOyeacGmpQk/giphy.gif?cid=ecf05e4780b2e3bfgdawfxb99ofvm0z47mbau8bhzjyz0trx&rid=giphy.gif&ct=g)